flowSteps/httpCall/step.json

```diff
@@ -0,0 +1,134 @@
+{
+	"label": "HTTP call",
+	"name": "httpCall",
+	"category": "integrations",
+	"description": "This flow step will send generic request",
+	"inputs": [
+		{
+			"label": "Method",
+			"name": "method",
+			"type": "choice",
+			"description": "Select method",
+			"defaultValue": "get",
+			"required": "true",
+			"options": {
+				"possibleValues": [
+					{
+						"label": "Get",
+						"name": "get"
+					},
+					{
+						"label": "Post",
+						"name": "post"
+					},
+					{
+						"label": "Put",
+						"name": "put"
+					},
+					{
+						"label": "Delete",
+						"name": "delete"
+					},
+					{
+						"label": "Connect",
+						"name": "connect"
+					},
+					{
+						"label": "Head",
+						"name": "head"
+					},
+					{
+						"label": "Options",
+						"name": "options"
+					},
+					{
+						"label": "Patch",
+						"name": "patch"
+					},
+					{
+						"label": "Trace",
+						"name": "trace"
+					}
+				],
+				"allowContextSelector": "false"
+			}
+		},
+		{
+			"name": "baseUrl",
+			"label": "Base URL",
+			"description": "The base URL where this endpoint will send request to. Add path variables in Url if required",
+			"type": "text",
+			"required": "true",
+			"options": {
+				"allowContextSelector": "false"
+			}
+		},
+		{
+			"name": "Headers",
+			"label": "headers",
+			"description": "The headers are JSON format, per example: {\"key\"=\"ABC123\",\"client\"=\"456789\".",
+			"type": "text",
+			"options": {
+				"allowContextSelector": "false"
+			}
+		},
+		{
+			"name": "Params",
+			"label": "params",
+			"description": "The params are JSON format, per example: {\"key\"=\"ABC123\",\"client\"=\"456789\".",
+			"type": "text",
+			"options": {
+				"allowContextSelector": "false"
+			}
+		},
+		{
+			"label": "Body",
+			"name": "body",
+			"type": "text",
+			"required": "['post', 'put', 'patch'].includes(config.method)",
+			"visibility": "['post', 'put', 'patch'].includes(config.method)",
+			"description": "This is used to set body request",
+			"options": {
+				"allowContextSelector": "false"
+			}
+		},
+		{
+			"label": "Requires callback",
+			"name": "requiredCallBack",
+			"type": "boolean",
+			"description": "If true it will return callback",
+			"defaultValue": false,
+			"options": {
+				"allowContextSelector": "false"
+			}
+		},
+		{
+			"label": "Callback data",
+			"name": "callbackData",
+			"type": "text",
+			"description": "This is used to set callback data",
+			"visibility": "config.requiredCallBack",
+			"options": {
+				"allowContextSelector": "false"
+			}
+		},
+		{
+			"label": "Callbacks",
+			"name": "callbacks",
+			"type": "text",
```

Comment:
Should be multiplicity many?

Reply:
Callback not support multiplicity many.
The following information provided in the documentation was used for [guidance](https://platform-docs.slingr.io/app_development_model_endpoints.html#callbacks).